From: Gioele Barabucci <gioele@svario.it>
Date: Sat, 20 Sep 2025 22:01:33 +0200
Subject: cmdline/terminal: Include `<cstdint>` for `UINT16_MAX`
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Without `<cstdint>` GCC 15 produces the following error:

```
../../../src/cmdline/terminal.cc: In member function ‘virtual unsigned int aptitude::cmdline::{anonymous}::terminal_impl::get_screen_width()’:
../../../src/cmdline/terminal.cc:143:20: error: ‘UINT16_MAX’ was not declared in this scope
  143 |             return UINT16_MAX;
      |                    ^~~~~~~~~~
```
Upstream-Status: Pending
See: https://bugs.debian.org/1096323
---
 src/cmdline/terminal.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cmdline/terminal.cc b/src/cmdline/terminal.cc
index 290097d..abaed7d 100644
--- a/src/cmdline/terminal.cc
+++ b/src/cmdline/terminal.cc
@@ -33,6 +33,7 @@

 #include <cwidget/generic/util/transcode.h>

+#include <cstdint>
 #include <iostream>
 #include <memory>

